tests/encodeInputSurface.cpp

```diff
@@ -193,7 +193,7 @@ bool EncodeInputSurface::getOneFrameInput(VideoFrameRawData& inputBuffer)
 {
     ANativeWindowBuffer* anb = NULL;
     memset(&inputBuffer, 0, sizeof(inputBuffer));
-    inputBuffer.memoryType = VIDEO_DATA_MEMORY_TYPE_ANDROID_NATIVE_BUFFER;
+    inputBuffer.memoryType = VIDEO_DATA_MEMORY_TYPE_ANDROID_BUFFER_HANDLE;
```

Comment:
We need check yami's api version first. Else it will break the build. I will send a patch to check YamiAPI version in configure.ac.